README.md

```diff
@@ -57,6 +57,8 @@ it needs to be disabled.
 Download the archive matching your host OS from the Minishift [releases page](https://github.com/minishift/minishift/releases) and unpack it. Then copy the contained binary to your preferred
 location and optionally ensure it is added to your _PATH_.
 
+**Note**: For Windows OS, you can run the binary from any location in the drive where your `%USERPROFILE%` directory is located. Mostly this is in the `C:` drive.
```

Comment:
Ok, I've seen now issue #236. I still think the wording is confusing. I suggest something along the lines of:

**Note**: Due to issue #236 (link), you need to execute the _minishift_ binary on Windows from the drive containing your %USERPROFILE% directory.